turbo/engineapi/engine_server.go

```diff
@@ -120,7 +120,7 @@ func (s *EngineServer) checkWithdrawalsPresence(time uint64, withdrawals []*type
 }
 
 // EngineNewPayload validates and possibly executes payload
-func (s *EngineServer) newPayload(ctx context.Context, req *engine_types.ExecutionPayload, version clparams.StateVersion) (*engine_types.PayloadStatus, error) {
+func (s *EngineServer) newPayload(ctx context.Context, req *engine_types.ExecutionPayload, expectedBlobVersionedHashes *[]libcommon.Hash, parentBeaconBlockRoot *libcommon.Hash, version clparams.StateVersion) (*engine_types.PayloadStatus, error) {
```

Comment:
`expectedBlobVersionedHashes` can probably be a slice rather than a pointer to slice since golang slices might be `nil`.